.github/README.md

```diff
@@ -0,0 +1,47 @@
+# [Oppia Android](https://www.oppia.org) ![CI Lint and Tests](https://github.com/oppia/oppia-android/workflows/CI%20Lint%20and%20Tests/badge.svg) [![Gitter](https://badges.gitter.im/oppia/oppia-android.svg)](https://gitter.im/oppia/oppia-android?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```

Comment:
1. Done.
2. This is intentionally on the same line as the title for consistency with https://github.com/oppia/oppia.
3. Is there an existing library for this? I like the idea of tracking them.
4. Sounds like a good idea--how would we do that?
5. Nice--done.

Reply:
3. Please check this - https://www.tickgit.com/
Adding this badge as an example. 
[![TODOs](https://img.shields.io/endpoint?url=https://api.tickgit.com/badge?repo=github.com/cilium/hubble)](https://www.tickgit.com/browse?repo=github.com/cilium/hubble)

4. We can go hardcode for now as we are migrating from gradle to bazel. Most of the solutions accept gradle file from which they extract the version and show it to the badge. 
[![Kotlin version badge](https://img.shields.io/badge/kotlin-1.2.31-9cf.svg)](http://kotlinlang.org/)
[![Ktlint version badge](https://img.shields.io/badge/Ktlint-0.37.0-9cf.svg)]()
[![bufbuild version badge](https://img.shields.io/badge/Bufbuild-v0.15.0-9cf.svg)]()
[![JDK version badge](https://img.shields.io/badge/Java-1.9-9cf.svg)]()
[![Android Studio version badge](https://img.shields.io/badge/Android%20Studio-4.1-9cf.svg)]()
[![Bazel version badge](https://img.shields.io/badge/Bazel-3.4.1-9cf.svg)]()

This is something hacktoberfest worthy to add. 
[![GitHub issues by-label](https://img.shields.io/github/issues/oppia/oppia-android/good%20first%20issue.svg)](https://github.com/oppia/oppia-android/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)

Reply:
we can only put effort on point 3 if we have time and you find it worthy, else we are ok to skip it.

Reply:
Do you see specific value in adding the versions? I'm not as inclined to if we can't do it automatically since theyu'll quickly become out of date.

I like the idea of the TODOs badge, but tickgit.com isn't working for me when I try to add oppia-android. Any ideas?

I like the good first issues badge link. I'll add that.

Reply:
I think the version think we can take after we completely remove the gradle, so in that we way we can have abadge which always get updated with the required bazel file for version.

Reply:
That sounds good to me. Does this look resolved to you then, @anandwana001?

Reply:
Yes